<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ringname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>铃声名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>75</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>日期时间：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnappend">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>添加铃声</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>75</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>铃声文件：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ringfile">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnchoose">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QListView" name="listname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listdt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listfile">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btndel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>删除铃声</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnplay">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>播放铃声</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtrepeat">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>每天</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtonce">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>单次</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnsort">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>时间排序</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbox6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>80</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>周六</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbox7">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>周日</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
